handbook/engineering/campaigns/index.md

```diff
@@ -41,7 +41,21 @@ Users provide the code to make the change, we provide the plumbing to turn it in
 
 ## Goals
 
-[WIP Goals (internal only)](https://docs.google.com/document/d/1ADmpfJuVhprHl8Eimuj4fMiVr1kCRVLZpt98HLr9618/edit)
+The short-term goal and primary focus of the team is to *deliver campaigns to all users*. Our milestones toward that goal:
+
+1. Someone on the Campaigns team makes one meaningful change to at least two repositories.
+1. A Sourcegraph engineer (not on this team) makes one meaningful change to at least two repositories.
+1. Reach out to one customer, summarizing the changes in the new Campaigns workflow, to give them a heads-up.
```

Comment:
I think we should do a variation of this as soon as we can: we need to let our customers that already use campaigns know about the new workflow and offer to walk them through it. The release is out and the previous workflow doesn't exist anymore.

Take a look at this issue https://github.com/sourcegraph/customer/issues/12 to get an idea of what the email could look like.

I also think it might be good to create such a customer issue, so I went ahead and created a rough draft with email drafts in it: https://github.com/sourcegraph/customer/issues/92

I assigned it to you, but if you don't feel comfortable sending those emails, I can also do it.

We should also probably ask @sqs and #analytics whether there are more customers we should consider reaching out to.